production/ksonnet/loki/shipper.libsonnet

```diff
@@ -8,48 +8,60 @@
   local containerPort = k.core.v1.containerPort,
 
   _config+:: {
+    // flag for tuning things when boltdb-shipper is current or upcoming index type.
+    using_boltdb_shipper: true,
+    using_tsdb_shipper: false,
```

Comment:
I think we can also enable this flag by default. Wdyt?

Reply:
I think its difficult to enable this by default as we also need to add schema config with a specific date, when they want to move to tsdb.

`
      schema_config: {
        configs: [{
          from: '2023-01-05',
          index: {
            period: '24h',
            prefix: 'loki_index_',
          },
          object_store: 'azure',
          schema: 'v12',
          store: 'boltdb-shipper',
        }, {
          from: '2023-06-27',
          index: {
            period: '24h',
            prefix: 'loki_index_',
          },
          object_store: 'azure',
          schema: 'v12',
          store: 'tsdb',
        }],
      },
`

Reply:
Fair point. I'm also ok to leave it as it.